exo.kernel.component.common/src/main/java/org/exoplatform/services/scheduler/impl/JobSchedulerServiceImpl.java

```diff
@@ -692,4 +715,20 @@ private String getGroupName(String initialGroupName)
          gname = containerName_ + ":" + initialGroupName;
       return gname;
    }
+
+   /**
+    * Return the list of existed jobs key
+    */
+   private void getAllJobKey() {
+      try {
+         List<String> jgroups = scheduler_.getJobGroupNames();
```

Comment:
The name is quite confusing :) We should rename it to jobGroupNames instead IMO.